source/reference/program/mongod.txt

```diff
@@ -156,8 +156,6 @@ Storage Options
 
 .. include:: /includes/option/option-mongod-nojournal.rst
 
-.. include:: /includes/option/option-mongod-journalOptions.rst
```

Comment:
I *think* Michael told me to kill it off, but I don't recall what the logic was, since it was a month ago. I'll plop 'er back in.